### PR TITLE
Transport config (client)

### PR DESCRIPTION
**Pull request by iaj2**
- #12

### CODE REVIEW

File: edgepi-leds.js [hunk 1]
```diff
@@ -7,9 +7,17 @@ module.exports = function(RED) {
...
-        const led = new rpc.LEDService();
+        let led = null;
+        let transport = null;
 
-        console.log(node.Method, node.LedPin)
+        if(config.transport === "Network"){
+            transport = `tcp://${config.tcpAddress}:${config.tcpPort}`;
+            led = new rpc.LEDService(transport); 
+        } else {
+            transport = 'ipc:///tmp/edgepi.pipe';
```
Comment (farzadpanahi) [inline edgepi-leds.js:17]: please define `'ipc:///tmp/edgepi.pipe'` as a constant

File: edgepi-leds.js [hunk 2]
```diff
@@ -7,9 +7,17 @@ module.exports = function(RED) {
...
+            led = new rpc.LEDService(transport);
+        }
```
Comment (farzadpanahi) [inline edgepi-leds.js:19]: I think you should be able to take `led = new rpc.LEDService(transport);` outside of if-else

**Review by farzadpanahi — CHANGES_REQUESTED** [review]

File: edgepi-leds.js (hunk 2 above)
Comment (iaj2) [reply edgepi-leds.js]: 63a226934951bf199ed7a8f4da11a89b006cfc62

File: edgepi-leds.js (hunk 1 above)
Comment (iaj2) [reply edgepi-leds.js]: 63a226934951bf199ed7a8f4da11a89b006cfc62

**Review by naomiven — APPROVED** [review]

**Review by farzadpanahi — APPROVED** [review]

**Pull request merged**